src/modules/menu/components/search/SearchResultListEntry.vue

```diff
@@ -75,6 +77,13 @@ export default {
                 this.$refs.item.tabIndex = '-1'
             }
         },
+        previewResult(show) {
+            if (this.resultType === 'layer') {
+                this.$emit('preview', show ? this.entry.layerId : null)
+            } else {
+                this.$emit('preview', show ? this.entry.coordinates : null)
+            }
+        },
```

Comment:
wouldn't be clearer (for us human) to fire a distinct event when the preview is done?
here only the payload tells us if it is ongoing or ended, I would maybe prefer to have a `preview-start` and `preview-stop` events if you don't mind

Reply:
Yeah. Makes sense! Like the `mouseenter` and `mouseleave` events.